bin/table_classification_summaries_main

```diff
@@ -18,6 +18,20 @@ from warmup.latex import format_median_error, get_latex_symbol_map
 from warmup.statistics import bootstrap_confidence_interval
 
 
+VM_NAMES_MAP = {
+    "JRubyTruffle": "JRuby+Truffle",
+    "Hotspot": "HotSpot"
+}
+
+BENCHMARK_NAMES_MAP = {
+    "binarytrees": "\\binarytrees",
+    "nbody": "\\nbody",
+    "fannkuch_redux": "\\fannkuch",
+    "richards": "\\richards",
+    "fasta": "\\fasta",
+    "spectralnorm": "\\spectralnorm",
+}
+
```

Comment:
Don't worry about this now, but this stuff would probably we better alongside `_MACHINES` in `warmup/krun_results.py`.